cmd/processesChecker_test.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	"github.com/stretchr/testify/assert"
+	"os"
+	"path"
+	"testing"
+)
+
+func Test_WarnMultipleProcesses(t *testing.T) {
+	a := assert.New(t)
+	err := os.Mkdir("temp", 0777) // Temp dir to simulate .azcopy dir
+	pidsDir := path.Join("temp", "pids")
+	defer os.RemoveAll("temp") // Cleanup
+	a.NoError(err)
+
+	err1 := WarnMultipleProcesses("temp", 1)
+	a.NoError(err1)
+
+	pid1 := path.Join(pidsDir, "1.pid")
+	_, err = os.Stat(pid1)
+	a.NoError(err, "first .pid file should exist")
+
+	// Act
+	err2 := WarnMultipleProcesses("temp", 2) // Additional AzCopy process, warning should output
```

Comment:
We could set up a mock LCM (I believe this already happens in the mock tests) to validate this functionality and make sure the expected warning gets output